hardware/asic/skywater/Makefile

```diff
@@ -40,7 +40,7 @@ $(DEFINE): sw boot.hex
 	cp boot.hex temp_src
 
 synth:  clean $(VSRC) $(VHDR) temp_src temp_inc $(DEFINE) 
-	cp /opt/OpenLane . 
+	cp -r /opt/OpenLane . 
```

Comment:
cannot have have hardcoded paths as users may use different paths

pls call it OPENLANE_SRC_DIR

Reply:
OPENLANE_SRC_DIR 
should it be inside Makefile only or it should be in bash?

Reply:
how can it be in makefile without being hardcoded?

Reply:
the user must define it -- write that in the readme

Reply:
export OPENLANE_SRC_DIR=<path to openlane src directory>
I have added to README
and this will be in bash now